evolver/calibration/demo.py

```diff
@@ -0,0 +1,17 @@
+from evolver.base import ConfigDescriptor
+from evolver.calibration.interface import Calibrator, Transformer
+
+
+class NoOpTransformer(Transformer):
+    def convert_to(self, data, *args, **kwargs):
+        return data
+
+    convert_from = convert_to
+
+
+class NoOpCalibrator(Calibrator):
+    class Config(Calibrator.Config):
+        input_transformer: ConfigDescriptor | Transformer | None = NoOpTransformer()
+        output_transformer: ConfigDescriptor | Transformer | None = NoOpTransformer()
+
+    def run_calibration_procedure(self, *args, **kwargs): ...
```

Comment:
With the addition of the ``LinearCalibrator``, etc, these demo classes seem quite redundant at this point - especially since they don't seem to demo much.

It seems that their only merit is their use in tests to de-tether breakages in ``polyfit`` causing cascading breakages in other unit tests, thus making them less unit-y.